Xamarin.Forms.Platform.Android/Renderers/ShellToolbarAppearanceTracker.cs

```diff
@@ -18,18 +18,19 @@ public ShellToolbarAppearanceTracker(IShellContext shellContext)
 		public virtual void SetAppearance(Toolbar toolbar, IShellToolbarTracker toolbarTracker, ShellAppearance appearance)
 		{
 			var foreground = appearance.ForegroundColor;
-			var background = appearance.BackgroundColor;
+			var backgroundColor = appearance.BackgroundColor;
+			var background = appearance.Background;
 			var titleColor = appearance.TitleColor;
 
-			SetColors(toolbar, toolbarTracker, foreground, background, titleColor);
+			SetColors(toolbar, toolbarTracker, foreground, backgroundColor, background, titleColor);
 		}
 
 		public virtual void ResetAppearance(Toolbar toolbar, IShellToolbarTracker toolbarTracker)
 		{
-			SetColors(toolbar, toolbarTracker, ShellRenderer.DefaultForegroundColor, ShellRenderer.DefaultBackgroundColor, ShellRenderer.DefaultTitleColor);
+			SetColors(toolbar, toolbarTracker, ShellRenderer.DefaultForegroundColor, ShellRenderer.DefaultBackgroundColor, ShellRenderer.DefaultBackground, ShellRenderer.DefaultTitleColor);
 		}
 
-		protected virtual void SetColors(Toolbar toolbar, IShellToolbarTracker toolbarTracker, Color foreground, Color background, Color title)
+		protected virtual void SetColors(Toolbar toolbar, IShellToolbarTracker toolbarTracker, Color foreground, Color backgroundColor, Brush background, Color title)
```

Comment:
We shouldn't modify this one because it's public

Can we create a brush version of this virtual?

protected virtual void SetColors(Toolbar toolbar, IShellToolbarTracker toolbarTracker, Brush foreground, Brush backgroundColor, Brush title)

Not sure if that will be too tricky to make non breaking from a behavior stand point but I think it works better then growing this method as we add brushes

Reply:
Changed, see the last comment added.